underactuated.html

```diff
@@ -6026,27 +6043,29 @@ <h1>Acrobots, Cart-Poles, and Quadrotors</h1>
 
     <section data-type="sect2"><h1>Download the binaries</h1>
 
-      <todo>Show linux OR mac (hide on click).</todo>
+      <div class="linux"><pre class="highlight"><code>curl -o drake.tar.gz <div id="drake-linux-binaries" style="display:inline">https://drake-packages.csail.mit.edu/drake/nightly/drake-latest-xenial.tar.gz</div></code></pre></div>
 
-      <p><b>Linux</b>
-      <div><pre class="highlight"><code>curl -o drake.tar.gz <div id="drake-linux-binaries" style="display:inline">https://drake-packages.csail.mit.edu/drake/nightly/drake-latest-xenial.tar.gz</div></code></pre></div>
-
-      <p><b>Mac</b></p>
-      <div><pre class="highlight"><code>curl -o drake.tar.gz <div id="drake-mac-binaries" style="display:inline">https://drake-packages.csail.mit.edu/drake/nightly/drake-latest-mac.tar.gz</div></code></pre></div>
+      <div class="mac"><pre class="highlight"><code>curl -o drake.tar.gz <div id="drake-mac-binaries" style="display:inline">https://drake-packages.csail.mit.edu/drake/nightly/drake-latest-mac.tar.gz</div></code></pre></div>
 
     </section>
 
     <section data-type="sect2"><h1>Unpack and set your PYTHONPATH</h1>
 
-      <div><pre class="highlight"><code>sudo tar -xvzf drake.tar.gz -C /opt
-export PYTHONPATH=/opt/drake/install/lib/python2.7/site-packages:${PYTHONPATH}</code></pre></div>
+      <div class="linux"><pre class="highlight"><code>sudo tar -xvzf drake.tar.gz -C /opt
+sudo /opt/drake/share/drake/setup/install_prereqs
+export PYTHONPATH=/opt/drake/lib/python2.7/site-packages:${PYTHONPATH}</code></pre></div>
 
+      <div class="mac"><pre class="highlight"><code>sudo tar -xvzf drake.tar.gz -C /opt
+/opt/drake/share/drake/setup/install_prereqs
+export PYTHONPATH=/opt/drake/lib/python2.7/site-packages:${PYTHONPATH}</code></pre></div>
     </section>
 
     <section data-type="sect2"><h1>Test for success</h1>
 
-      <div><pre class="highlight"><code>python -c 'import pydrake; print(pydrake.__file__)'
-</code></pre></div>
+      <p class="mac">NOTE: You must ensure that you have Homebrew Python installed, and are <a href="http://drake.mit.edu/python_bindings.html#using-the-python-bindings">using python2 from Homebrew Python</a> to execute these scripts, otherwise you python interpreter will crash in the test below.</p>
```

Comment:
`you python interpreter` -> `your Python interpreter`

Do you want to suggest `export PATH="/usr/local/opt/python/libexec/bin:${PATH}"`?

Reply:
maybe that's cleaner.  done.